ucan-invocation/functions/space-metrics-store-remove-total.js

```diff
@@ -0,0 +1,52 @@
+import * as Sentry from '@sentry/serverless'
+
+import { createSpaceMetricsTable } from '../tables/space-metrics.js'
+import { parseKinesisEvent } from '../utils/parse-kinesis-event.js'
+
+Sentry.AWSLambda.init({
+  environment: process.env.SST_STAGE,
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
+})
+
+const STORE_REMOVE = 'store/remove'
```

Comment:
These should come from `@web3-storage/capabilities` so that if they change we'll know here.